MAUI/Autocomplete/UI-Customization.md

```diff
@@ -859,3 +859,58 @@ autocomplete.CursorPosition = 4;
 {% endtabs %}
 
 N> Two way binding not supported in Android platform.
+
+## ShowBorder
+
+The `ShowBorder` property is used to modify the visibility of border. The following code example demonstrates how to change the border visibility in [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html). The default value of the `ShowBorder` property is **true**.
```

Comment:
please modify the content,

The `ShowBorder` property is used to modify the visibility of the border and its default value is **true**. The following code example demonstrates how to change the border visibility in [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html).